sys/random/minstd.c

```diff
@@ -43,7 +38,7 @@ int rand_minstd(void)
 {
```

Comment:
The return value should be `int32_t` as well.

Reply:
Sure... I amended the change to the first commit.